d3a_api_client/utils.py

```diff
@@ -276,14 +278,17 @@ def log_bid_offer_confirmation(message):
 
 def get_simulation_info(simulation_id=simulation_id_from_env, domain_name=domain_name_from_env,
                         websockets_domain_name=websocket_domain_name_from_env):
-    if os.environ['JSON_FILE_PATH'] != "":
+    if os.environ.get('JSON_FILE_PATH', "") != "":
         with open(os.environ['JSON_FILE_PATH']) as json_file:
             simulation_info = json.load(json_file)
-            simulation_id = simulation_info['uuid']
-            domain_name = simulation_info['domain_name']
-            websockets_domain_name = simulation_info['web_socket_domain_name']
+        if set(simulation_info.keys()) != {"username", "name", "uuid", "domain_name",
+                                           "web_socket_domain_name", "global_settings", "registry"}:
+            raise SimulationInfoException(f"Simulation_file doesn't have all the keys")
```

Comment:
Please be consistent in the naming. Please change 
- `Simulation_file` here in the error message
- `SimulationInfoException`
- `api-client-summary` in the resources folder
to the proposal in Spyros comment (or different, as long as they are the same)
Thanks!